### PR TITLE
Mykola shydlovsky.practice 2

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/components/Form/Form.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    onCheck = () => { /*This function look on value our checkBox and set new set*/ /*We need use arrow function beacouse context*/
+        const { isChecked } = this.state;
+
+        this.setState({
+            isChecked: !isChecked
+        });
+    };
+
+    o/*nFormSubmit = () => {
```
Comment (AlyonaPianykh) [inline src/components/Form/Form.js]: You don't need this code here

File: src/components/Toggle/Toggle.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        super(props);
+    }
+
+    render() {
+
+        const {type = 'checkbox', isToggleChecked, onChange} = this.props;
+
+        return (
+            <label className="toggle-wrap">
+                <div>{this.props.label}</div>
```
Comment (AlyonaPianykh) [inline src/components/Toggle/Toggle.js]: Please destruct `label ` from  this.props in code above

File: src/components/Form/Form.js (hunk 1 above)
Comment (MikolaLvivUA) [reply src/components/Form/Form.js]: Done

File: src/components/Toggle/Toggle.js (hunk 2 above)
Comment (MikolaLvivUA) [reply src/components/Toggle/Toggle.js]: Done

**Pull request closed without merging**